js/helpDialog.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Creates a <div> Element with helpful hints, and returns it for appending to the "level" element
+ */
+
+define(() => {
+	const helpDialog = document.createElement("div");
+	helpDialog.id = "help-dialog";
+	helpDialog.className = "help-dialog";
+
+	const heading = document.createElement("h3");
+	heading.className = "help-dialog__heading";
+	heading.innerHTML = "Hint:";
+
+	const hint = document.createElement("p");
+	hint.className = "help-dialog__hint";
+	hint.innerHTML =
+		"Whenever you click on a square, it will change to the first available color of alien. If you click it again it will change to a different color, but ONLY if it doesn't cause there to be three of the same color in a row, column, or diagonal.<br/><br/> When you click on a square, if that square would have caused three in a row, then a different color alien will appear, unless there is no valid color available. If there is no valid move, the square will shake.";
```

Comment:
I feel like this text is too technical. It's a message for the player, not for the engineer, because this feels more like documentation about the color picking logic more than how the game works.

Maybe this text should just explain that you can't have more than 3 aliens alligned and some other usefull information?

Reply:
Heard. 

When I first started playing the game, it wasn't clear _how_ a specific color was chosen, which was the initiative for the help dialog. 

What about something super simple like: 

"If you click a square and you'd like to change the color, click it again and the alien's color will change. If it doesn't change, no other color is valid"

Reply:
Looking great! (Sorry for the late response)